assets/scss/components/elements/_content.scss

```diff
@@ -58,6 +58,7 @@ pre {
 .excerpt-wrap {
 	--listpad: 20px;
 	--liststyle: disc;
+	--contentimgmargin: 1em;
```

Comment:
I think we should use `px` instead of `em` as most sizes in the new skin are defined using `px` instead of `em`. I would suggest something like `12px` or `24px`. What do you think?

Reply:
Thank you, @preda-bogdan! 24px it is 🚀